cfgov/preprocessed/css/organisms/sidebar-breakout.less

```diff
@@ -0,0 +1,152 @@
+                                    content-l_col-2-3">
+                        <m-text-introduction></m-text-introduction>
+                    </section>
+                    <aside class="o-sidebar-breakout_col
+                                  content-l_col
+                                  content-l_col-1-3
+                                  block
+                                  block__flush-top
+                                  block__flush-bottom">
+                        <h3 class="o-sidebar-breakout_heading">
+                        </h3>
+                        <div class="o-sidebar-breakout_img-container
+                                    u-centered-on-mobile">
+                            <div class="o-sidebar-breakout_img
+                                        u-centered-on-mobile"
+                                 style="background-image:url()">
+                            </div>
+                        </div>
+                        <div class="o-sidebar-breakout_text-container">
+                            <h4 class="o-sidebar-breakout_text-heading">
+                                <span class="cf-icon"></span>
+                            </h4>
+                            <p class="o-sidebar-breakout_text-body short-desc">
+                            </p>
+                            <m-related-post></m-related-posts>
+                        </div>
+                    </aside>
+                </div>
+            </div>
+        s</div>
+        <div class="content_line"></div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .o-sidebar-breakout
+            .content_intro
+              .content-l
+                .o-sidebar-breakout_col
+                    .m-text-introduction
+                .o-sidebar-breakout_col
+                  .o-sidebar-breakout_heading
+                  .o-sidebar-breakout_img-container
+                    .o-sidebar-breakout_img
+                  .o-sidebar-breakout_text-container
+                    .o-sidebar-breakout_text-heading
+                      .cf-icon
+                    .o-sidebar-breakout_text-body
+                    .m-related-posts
+          .content_line
+  tags:
+    - cfgov-organisms
+*/
+
+@margin__em: unit( @grid_gutter-width / @base-font-size-px, em );
+@image_width__px:  300px;
+@image_height__px: 170px;
+
+.o-sidebar-breakout {
+    .wrapper();
+
+    .respond-to-max(@mobile-max, {
+        margin-bottom: @margin__em;
+    });
+
+    &_col {
+
+        .respond-to-max(800px, {
+            width: 100%;
+            &:last-child {
+                padding-top: @margin__em;
+                border-top: 1px solid @gray-50;
+            }
+        });
+    }
+
+    &_heading {
+        margin-bottom: @margin__em;
+    }
+
+    &_img {
+        background-repeat: no-repeat;
+        background-size: contain;
+        padding-top: 56.25%;
```

Comment:
fixed in https://github.com/cfpb/cfgov-refresh/commit/0152b8b34825bacf002abfe0394409cf6e87bba3

![magic](https://cloud.githubusercontent.com/assets/1696212/10997433/1934bdec-845b-11e5-80a3-fff480fc9371.gif)